.travis.yml

```diff
@@ -57,32 +58,39 @@ jobs:
         - export ARCH=x86_64 KCFLAGS="-Wall -Werror"
         - export MAKEFLAGS=j"$(nproc)"
         - bash kconfig/kconfig-sof-default.sh
-        - make modules_prepare
         - make sound/ W=1
         - make drivers/soundwire/ W=1
         - make
         - make bindeb-pkg
     - name: "BUILD GCC9 SST Kernel x86_64"
       script:
-        - export ARCH=x86_64
+        - export ARCH=x86_64 KCFLAGS="-Wall -Werror"
```

Comment:
If those are shared ENV we can make them into global env
https://docs.travis-ci.com/user/environment-variables/
```
env:
  global:
    - ARCH=x86_64
    - KCFLAGS="-Wall -Werror"
    - MAKEFLAGS=j"$(nproc)"
```

Reply:
no, this has to be done on a target basis.